### DISCUSSION TITLE
Native UI elements for settings & menu

### DISCUSSION THREAD

**Discussion by ealitten** [mozilla-mobile/firefox-ios#16313] [Ideas]
I have both Firefox and Firefox Focus installed, and I much prefer the UI in FF Focus - it looks much better and more consistent with other iOS apps than regular FF, which I assume is because native UI elements have been used. I'm wondering if there is any plan to bring this across from Focus to regular Firefox? 

If we look at the settings page, in Focus it appears to be using native controls for the toggles etc, and is more visually consistent with the platform (i.e. iOS):

![IMG_3570](https://github.com/mozilla-mobile/firefox-ios/assets/19875457/f1b9e9f6-1e9d-4ca0-bd19-cacb9a4fbdec)

![IMG_3569](https://github.com/mozilla-mobile/firefox-ios/assets/19875457/864b78b8-c6fc-473b-9daf-9381aa1277bb)

A similar story for the menu - in Focus we have a native menu which closely matches the styling in Safari iOS, whereas Firefox has a clearly custom menu (which also has more spacing and struggles to fit on screen):

![IMG_3567](https://github.com/mozilla-mobile/firefox-ios/assets/19875457/ad214fd4-13e0-4c45-9d91-b1d8a915cad0)

![IMG_3568](https://github.com/mozilla-mobile/firefox-ios/assets/19875457/0cb31c9f-1c4d-41c8-8d58-596597042b04)

Safari menu for comparison:

![IMG_3571](https://github.com/mozilla-mobile/firefox-ios/assets/19875457/1e73c7c9-4f95-4f78-9b2b-7a5d3f106ce9)

(Edit: make images smaller)

**Comment by lmarceau** [comment]
There was some discussion about it if I remember correctly, I don't think we have a feature-request opened for this thought. Do you want to open one [here](https://github.com/mozilla-mobile/firefox-ios/issues/new?assignees=&labels=Feature-Request&projects=&template=---feature.md)?

**Reply by ealitten** [reply]
Thanks for the reply. I have opened one here: https://github.com/mozilla-mobile/firefox-ios/issues/16335
